dds_web/api/project.py

```diff
@@ -392,7 +390,9 @@ def post(self):
                 else:
                     # If it is an existing user, add them to project.
                     add_user_result = AddUser.add_user_to_project(
-                        existing_user=existing_user, project=new_project.public_id, owner=owner
+                        existing_user=existing_user,
+                        project=new_project.public_id,
+                        owner=user.get("role") == "Project Owner",
```

Comment:
Is the `add_user_to_project` code adapted to if the new added user has role Unit Admin or Personnel? I think there needs to be a few checks here before the user is added, and there needs to be a way of just adding the user to the unit if it's not a Researcher or Personnel.

Reply:
I've added a check for 'Project Owner' and 'Researcher' when adding users to a project. I think that adding users to a Unit needs to be a different function and not mixed in with the project addition functionality.

Reply:
Great! Would be great if we could add some tests for when attempting to add Unit Personnel / Admins, to check that it fails.

Reply:
Added a few tests!

Reply:
Nice! Thanks